website/sidebars.js

```diff
@@ -11,7 +11,37 @@
 
  module.exports = {
   docsSidebar: [
-    'Getting-Started',
+    {
+      type: 'category',
+      label: 'Getting Started',
+      items: [
```

Comment:
This looks great. While the autogen.net is also coming soon, maybe it's also worth considering provide dotnet implementation for these cases?

Reply:
Right. I think we can create a .NET version of this tutorial and add links to each chapter pointing to the .NET version of the chapter.

Reply:
Do you want to start writing a page in the .NET doc about the conversation patterns? Once that is done we can add the links. Overtime once we have matching examples and design patterns, we can make tabbed switching between the Python and .NET sample code.

Reply:
Sure I can add these conversation patterns to dotnet website once I finish the message refactor PR